orangecontrib/associate/widgets/owassociate.py

```diff
@@ -35,6 +35,7 @@ class Error(widget.OWWidget.Error):
     class Warning(widget.OWWidget.Warning):
         cont_attrs = widget.Msg("Data has continuous attributes which will be skipped.")
         err_reg_expression = widget.Msg("Error in {} regular expression: {}")
+        filter_no_match = widget.Msg("Filtering yielded no results.")
```

Comment:
Hm, it yielded a result, but it's empty. Maybe better: "No rules match the filter" or sth like this?

Reply:
Fixed.